File: src/test/java/TC/연속된수의합Test.java
package TC;

import org.assertj.core.api.Assertions;
import org.example.TC.연속된수의합;
import org.junit.jupiter.api.Test;

public class 연속된수의합Test {

    private final 연속된수의합 pro = new 연속된수의합();

    @Test
    void solution() {
        int[] result = pro.solution(4, 14);
        int[] answer = {2,3,4,5};
        Assertions.assertThat(result).isEqualTo(answer);
    }

}

File: src/test/java/TC(K)/연속된수의합Test.kt
package `TC(K)`

import io.kotest.core.spec.style.StringSpec
import org.assertj.core.api.Assertions
import org.example.TCK.연속된수의합

class 연속된수의합Test:StringSpec (
    {
        "연속 된 수의 합 계산" {
            val num:Int = 3
            val total:Int = 12
            val answer: IntArray = intArrayOf(3,4,5)
            val result:IntArray = 연속된수의합().solution(num, total)
            Assertions.assertThat(answer).isEqualTo(result)
        }
        "연속 된 수의 합 계산2" {
            val num:Int = 3
            val total:Int = 12
            val answer: IntArray = intArrayOf(3,4,5)
            val result:IntArray = 연속된수의합().solution2(num, total)
            Assertions.assertThat(answer).isEqualTo(result)
        }
    }
        )

